docs/network-flow-visibility.md

```diff
@@ -111,7 +111,11 @@ Please note that the default value for `flowCollectorAddr` is `"flow-aggregator.
 which uses the DNS name of the Flow Aggregator Service, if the Service is deployed
 with the Name and Namespace set to `flow-aggregator`. If you deploy the Flow Aggregator
 Service with a different Name and Namespace, then either use the appropriate DNS
-name or the Cluster IP of the Service. Please note that the default values for
+name or the Cluster IP of the Service.
+
+Please note that for Antrea Agent running on a Windows node, `flowCollectorAddr` can only be IP right now because there is a DNS resolution issue in current Windows support.
```

Comment:
Merge this line with the above paragraph. Something like..

"... with the Name and Namespace set to `flow-aggregator`. For Antrea Agent running on a Windows node, the user is required to change the default value of `HOST` in `flowCollectorAddr` from DNS name to the Cluster IP of Flow Aggregator service. The reason is because of the DNS resolution issue in the current Windows support of Kubernetes. In addition, if you deploy the Flow Aggregator Service with a different Name and Namespace, then either use the appropriate DNS name or the Cluster IP of the Service."

Reply:
Thanks, addressed.